### DISCUSSION TITLE
Only surport node >= 16

### DISCUSSION THREAD

**Discussion by fjloser** [unocss/unocss#3118] [General]
My NodeJs is 14 version and it can't right work，i use  latest unocss  version，Do I have to upgrade node

**Comment by antfu** [comment]
I'd happy to fix it, but actually even node 16 just reached EOL, so I am not sure if we should continue concerned about node 14. Instead, I think we could update the docs to make clear of the supported version range

**Reply by chizukicn** [reply]
I am more curious about the minimum environmental requirements for unocss to run on the browser. For example, do we need to consider IE 11?

**Reply by antfu** [reply]
IE is dead, isn't it? I don't see any reason to support it.

If you mean for the CSS, it really depends on each rule. If you mean the UnoCSS runtime, I think we might need to do some tests but I imagine all modern browsers would work.

**Reply by chizukicn** [reply]
😊 Thanks! I'm referring to @unocss/runtime and the core and presets it depends on. We really don't need to consider IE11.

**Reply by mdxiaohu** [reply]
@antfu 
Can I support version 14 .
Upgrading nodes to old projects can cause many problems .
